addon/components/high-charts.js

```diff
@@ -7,10 +7,11 @@ export default Ember.Component.extend({
   mode:    undefined,
   chartOptions: undefined,
   chart: null,
+  theme: undefined,
 
   buildOptions: Ember.computed('chartOptions', 'content.@each.isLoaded', function() {
     var chartContent, chartOptions, defaults;
-    chartOptions = this.getWithDefault('chartOptions', {});
+    chartOptions = jQuery.extend(true, this.get('theme'), this.get('chartOptions'));
```

Comment:
You should use `jQuery.extend(true, {}, this.get('theme'), this.get('chartOptions'))`. Otherwise, it will merge `chartOptions` into `theme`, which isn't the desired effect.